docs/aca/05-aca-dapr-pubsubapi/index.md

```diff
@@ -374,23 +404,40 @@ What we've done In the code above is the following:
 }
 ```
 {: .note }
-It is optional to use SendGrid APIs to send emails, you can use another service to complete the workshop or you can just simulate sending email by returning always `true` from the `SendEmail` method. [Next module](/aca/06-aca-dapr-bindingsapi/index.md/#1-create-dapr-sendgrid-output-binding-component-file) we will be using a new type of Dapr components to send emails using SendGrid.
+If you don't want to bother with signing up for a SendGrid account to send emails, you can just simulate sending emails by returning always `ok` from the `TaskSaved` method as shown below. In the [next module](/aca/06-aca-dapr-bindingsapi/index.md/#1-create-dapr-sendgrid-output-binding-component-file) we will be using a new type of Dapr components to send emails using SendGrid.
```

Comment:
what if they want to setup Sendgrid at this step? Should we add link to create account etc? because even in next module to simulate real sending of email, they will require to setup SendGrid account

Reply:
Good call. I will address that.

Reply:
Done